src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Formatting/HtmlFormattingPass.cs

```diff
@@ -72,15 +73,15 @@ public async override Task<FormattingResult> ExecuteAsync(FormattingContext cont
             return new FormattingResult(finalEdits);
         }
 
-        private List<TextChange> AdjustRazorIndentation(FormattingContext context, int startLine, int endLine)
+        private List<TextChange> AdjustRazorIndentation(FormattingContext context)
         {
             // Assume HTML formatter has already run at this point and HTML is relatively indented correctly.
             // But HTML doesn't know about Razor blocks.
             // Our goal here is to indent each line according to the surrounding Razor blocks.
             var sourceText = context.SourceText;
             var editsToApply = new List<TextChange>();
 
-            for (var i = startLine; i <= endLine; i++)
+            for (var i = 0; i < sourceText.Lines.Count; i++)
```

Comment:
This was necessary for cases when Html formatter wouldn't change a line because it thinks it is in the right position but we know it should have a different indentation.